docker/local/fetch_and_run.sh

```diff
@@ -0,0 +1,128 @@
+#!/bin/bash
+
+# Source https://github.com/awslabs/aws-batch-helpers/blob/master/fetch-and-run/fetch_and_run.sh
+# Copyright 2013-2017 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You may not use this file except in compliance with the
+# License. A copy of the License is located at
+#
+# http://aws.amazon.com/apache2.0/
+#
+# or in the "LICENSE.txt" file accompanying this file. This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES
+# OR CONDITIONS OF ANY KIND, express or implied. See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This script can help you download and run a script from S3 using aws-cli.
+# It can also download a zip file from S3 and run a script from inside.
+# See below for usage instructions.
```

Comment:
May we add https://aws.amazon.com/blogs/compute/creating-a-simple-fetch-and-run-aws-batch-job/ to this PR description for how to use the script here?

Reply:
Sorry, I see it is already in the description. I will mark it as resolved.